src/pages/style.css

```diff
@@ -115,18 +115,8 @@
 }
 
 /*
-***
-* DevDot
-***
-* The custom media and classes below are helpers used in DevDot.
-***
+Mobile & tablet visibility helpers, specific to dev-dot.
 */
-
-@custom-media --dev-dot-mobile only screen and (max-width: 728px);
```

Comment:
Moved these to a dedicated `custom-media` file, which we now `importFrom` in `postcss.config.js`. I think this will make these more reliably available. (Honestly not exactly sure how they're available as-is).